azure/azure-how-to/instances/find-ubuntu-images.rst

```diff
@@ -9,6 +9,26 @@ Free long term support offers
 
 To learn more about LTS versions of Ubuntu read `this article <https://ubuntu.com/blog/what-is-an-ubuntu-lts-release>`_.
 
+Ubuntu 24.04 LTS - Noble Numbat
+++++++++++++++++++++++++++++++++++
+
+| Quick start: `24.04 LTS on Azure <https://portal.azure.com/#create/canonical.ubuntu-24_04-ltsserver>`_
+| Quick start (China): `24.04 LTS on Azure China <https://portal.azure.cn/#create/canonical.canonical.ubuntu-24_04-ltsserver>`_
+
+.. list-table::
+   :widths: 19 50
+   :header-rows: 1
+
+   * - **Kind**
```

Comment:
Given that the first column always contains the same two pieces of information and the word "Kind" is ambiguous, would it make sense to use three columns labelled "Architecture", "Hyper-V Generation", and "URN"? The same applies to the other two table instances below.

Reply:
Sure but this would mean changing the entire page (to keep things consistent)

Reply:
https://github.com/canonical/ubuntu-cloud-docs/pull/207